x-pack/plugins/security/server/authentication/http_authentication/http_authorization_header.ts

```diff
@@ -33,7 +33,7 @@ export class HTTPAuthorizationHeader {
       return null;
     }
 
-    const [scheme] = authorizationHeaderValue.split(/\s+/);
+    const [scheme] = authorizationHeaderValue.split(/\s+/, 1);
```

Comment:
**note** not strictly required, but I found this while searching our plugin code. Limiting the split is more performant when we only care about a fixed number of tokens. I can revert this if you'd prefer